app/src/main/resources/ui/helper-foreign-keys.js

```diff
@@ -106,6 +112,10 @@ async function createForeignKeyLinks(index, linkType) {
     let addLinkForeignKeyButton = createButton(`add-foreign-key-${linkType}-link-btn-${index}`, "add-link", "btn btn-secondary", "+ Link");
     addLinkForeignKeyButton.addEventListener("click", async function () {
         numForeignKeysLinks += 1;
+        if (linkSourceForeignKeys.childElementCount > 1) {
```

Comment:
Shouldn't this check `>= 1` since if a link already exists, then we add in a divider before adding in the second foreign key link.